SignalServiceKit/src/Messages/Interactions/TSOutgoingMessage.m

```diff
@@ -781,8 +781,7 @@ - (OWSSignalServiceProtosDataMessageBuilder *)dataMessageBuilder
     if (!attachmentWasGroupAvatar) {
         NSMutableArray *attachments = [NSMutableArray new];
         for (NSString *attachmentId in self.attachmentIds) {
-            NSString *_Nullable sourceFilename = self.attachmentFilenameMap[attachmentId];
```

Comment:
This is the change I was least confident with - but it seems like this attachmentFileName map is redundant with what's already living on TSAttachmentStream.